src/core/modules/StarMgr.cpp

```diff
@@ -680,6 +681,14 @@ void StarMgr::populateHipparcosLists()
 				QMap<StelObjectP, float> sa;
 				sa[so] = static_cast<float>(getGcvsPeriod(s->getHip()));
 				variableHipStars.push_back(sa);
+				
+				auto vartype = getGcvsVariabilityType(s->getHip());
+				if (vartype.contains("EA"))
+				{
+					QMap<StelObjectP, float> sal;
```

Comment:
This part of code may be written more compact

Reply:
sorry, I don't see what you mean?